pyseq.py

```diff
@@ -876,6 +1060,40 @@ def get_sequences(source):
             seq = Sequence([item])
             seqs.append(seq)
 
-    log.debug('time: %s' % (datetime.now() - start))
-
-    return list(seqs)
+    log.debug("time: %s" %(datetime.now() - start))
+    if not stereo:
+        log.info("added sequences: %s" %(seqs))
+        return seqs
+    else:
+        seqs.sort()
```

Comment:
to explain a bit further I had this "stereo" keyword in there to let one decide if you are chasing paired sequences or not..
still it can be True all the time and if a sequence is not having a view and cannot be paired the normal sequence object will still be returned...